docs/architecture/cloud-native/definition.md

```diff
@@ -15,46 +16,46 @@ Consider the [definition](https://github.com/cncf/foundation/blob/master/charter
 
 As you can see, eBay, Netflix and Uber expose systems that consist of hundreds of self-contained isolated microservices. Impressively, they can rapidly respond to market conditions by instantaneously updating specific functionality without having to fully deploy the entire system.
 
-At its core, this agility derives from the architecture and infrastructure upon which the system is constructed, which include the following six capabilities, as shown below in Figure 1-2.
+At its core, this agility derives from the architecture and infrastructure upon which the system is constructed and is composed of six capabilities shown in Figure 1-2.
 
 ![Capabilities of cloud native apps](media/capabilities-of-cloud-native-apps.png)
 **Figure 1-2**. Capabilities of cloud native apps
 
-As each of these capabilities are key to a cloud native system, let’s take some time to visit each.
+As these capabilities are key to a cloud native system, let’s take some time to visit each.
 
 ## Thinking in terms of the cloud…
 
-Cloud native systems are *all about the cloud*: Born, developed and deployed in the cloud – specifically designed to thrive in a cloud environment. Development teams design these systems from the ground-up in a highly-dynamic, virtualized environment. They are built to exploit the platform and infrastructure capabilities of the cloud delivery model: Elasticity, resilience, redundancy, orchestration and on-demand features. 
+Cloud native systems are *all about the cloud*: Designed, developed, and deployed to thrive in a cloud environment. Development teams design these systems from the ground-up in a highly-dynamic, virtualized environment. They are built to exploit the platform and infrastructure capabilities of the cloud delivery model: Elasticity, resilience, redundancy, orchestration, and on-demand features. 
 
-The focus moves to “*how”* the application is created and deployed, not so much “*where*.” Cloud native apps make extensive use of PaaS (Platform as a Service) infrastructure and managed services. 
+The focus moves to *how* the application is created and deployed, not so much *where*. Cloud native apps make extensive use of PaaS (Platform as a Service) infrastructure and managed services. 
 
-Figure 1-3, shown below depicts the range of the full-featured compute services available in Azure. As you move from the top of the hierarchy to the bottom, your focus shifts away infrastructure plumbing to business functionality. Cloud native systems typically reside at the lower levels of the stack leveraging containers, PaaS and Serverless environments.
+Figure 1-3 depicts the range of the full-featured compute services available in Azure. Moving from the top of the hierarchy to the bottom, your focus shifts from infrastructure plumbing to business functionality. Cloud native systems typically reside at the lower levels of the stack leveraging containers, PaaS, and Serverless environments.
```

Comment:
I think the original was better "As you move..."

Reply:
Fixed